connect/src/com/telenor/connect/utils/HeadersDateUtil.java

```diff
@@ -0,0 +1,39 @@
+package com.telenor.connect.utils;
+
+import android.annotation.SuppressLint;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+import retrofit.client.Header;
+
+public class HeadersDateUtil {
+
+    @SuppressLint("SimpleDateFormat")
+    private final static SimpleDateFormat DATE_FORMAT
+            = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz");
+
+    @Nullable
+    public static Date get(@NonNull List<Header> headers) {
+        try {
+            return getDate(headers);
+        } catch (ParseException e) {
+            e.printStackTrace();
```

Comment:
I agree that the exception should not be propagated out. However: Is it normal to print stack traces to stderr in this repo specifically and in Android apps in general? Is there no application log? (Sorry about my ignorance)